lib/bootloader_setup.pm

```diff
@@ -307,8 +308,8 @@ sub bootmenu_type_console_params {
     type_string_very_slow "console=${serialdev}${baud_rate} ";
 
     # See bsc#1011815, last console set as boot parameter is linked to /dev/console
-    # and doesn't work if set to serial device. Don't want this on ipmi backend.
-    type_string_very_slow "console=tty " unless check_var('BACKEND', 'ipmi');
+    # and doesn't work if set to serial device. Don't want this on remote backends.
+    type_string_very_slow "console=tty " unless (is_remote_backend);
```

Comment:
or maybe we can just remove ``check_var('BACKEND', 'svirt')`` from ``is_remote_backend``

Reply:
Or maybe find what is the actual condition?

Reply:
is_remote_backend is already used in multiple places, so changing that will affect some backends =(

Reply:
@rwx788 but not many:
```
$ git grep is_remote_backend
lib/bootloader_setup.pm:use version_utils qw(is_caasp is_jeos is_leap is_sle is_remote_backend);
lib/bootloader_setup.pm:    type_string_very_slow "console=tty " unless (is_remote_backend);
lib/main_common.pm:    if (get_var("ISO_MAXSIZE") && (!is_remote_backend() || is_svirt_except_s390x())) {
lib/opensusebasetest.pm:use version_utils qw(is_sle is_leap is_upgrade is_aarch64_uefi_boot_hdd is_remote_backend);
lib/opensusebasetest.pm:    wait_serial(get_login_message()) unless is_remote_backend;
lib/version_utils.pm:          is_remote_backend
lib/version_utils.pm:          is_remote_backend
lib/version_utils.pm:sub is_remote_backend {
```

Reply:
--> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6066

Reply:
svirt is a remote backend, so it's wrong first of all because it is remote backend. Logic regarding "console=tty" relies on something else and not just if backend is remote or not, so create another method called e.g. `is_using_console_tty` and put specific condition there, or fix inline as it will be single usage. Our assumption that this logic depends on if backend is remote or not was wrong, simple as that.